src/stability.md

```diff
@@ -72,6 +68,22 @@ even on an `unstable` function, if that function is called from another
 Furthermore this attribute is needed to mark an intrinsic as callable from
 `rustc_const_stable` functions.
 
+## Stabilizing a library feature
+
+To stabilize a feature, follow these steps:
+
+1. Change `#[unstable]` to `#[stable(since = "version")]`.
+   `version` should be the *currently nightly*, i.e. stable+2. You can see which version is
+   currently nightly [on Forge](https://forge.rust-lang.org/#current-release-versions).
+2. Remove `#![feature]` from any test or doc-test for this API. If the feature is used in the
+   compiler or tools, remove it from there as well.
+3. If applicable, change `#[rustc_const_unstable]` to `#[rustc_const_stable(since = "version")]`.
+4. Open a PR against `rust-lang/rust`
+   - Add the appropriate labels: `@rustbot modify labels: +T-libs +needs-fcp`.
+   - Include a description of the feature and why it should be stabilized.
```

Comment:
Oh ... that seems really unfortunate, that means that only team members can get the process moving for stabilization. I like stabilization PRs because anyone can make them (e.g. https://github.com/rust-lang/rust/pull/80011).

Reply:
Is sending a PR for stabilization really much different than just asking a libs team member to start the FCP on the tracking issue? Either way, a libs team member gets pinged and needs to kick off the rfcbot.

We've had a lot of discussions split up over the tracking issue and stabilization PR lately. And some of the stabilization PRs have stayed open for a long time while this discussion went on such that the submitter kept having to fix merge conflicts and update the `since = ..` attribute.

Reply:
Fair enough, it's a T-libs decision to make anyway.